README.md

```diff
@@ -63,26 +63,21 @@ pytest -q -s test_flash_attn.py
 
 
 ## Installation and features
-
-Requirements:
-- CUDA 11.6 and above.
+**Requirements:**
+- A GPU backend (CUDA or ROCm)
```

Comment:
Let's phrase it as "CUDA toolkit or ROCm toolkit" to avoid people thinking you need a GPU to install. E.g. you just need nvcc, not the GPU itself.

Reply:
Done. Also added "using CUDA toolkit" in reference to the `ninja` compile times. AMD compile times are still pretty long even with `ninja`.